src/geometry/manifold/cont_mdiff.lean

```diff
@@ -2121,3 +2121,68 @@ hf.smul hg
 lemma smooth.smul {f : M → 𝕜} {g : M → V} (hf : smooth I 𝓘(𝕜) f) (hg : smooth I 𝓘(𝕜, V) g) :
   smooth I 𝓘(𝕜, V) (λ p, f p • g p) :=
 hf.smul hg
+
+/-! ### Smoothness of an open embedding and its inverse -/
+
+lemma cont_mdiff_open_embedding
+  {𝕜 : Type*} [nontrivially_normed_field 𝕜]
+  {E : Type*} [normed_add_comm_group E] [normed_space 𝕜 E]
+  {H : Type*} [topological_space H] (I : model_with_corners 𝕜 E H)
+  {M : Type*} [topological_space M] [nonempty M]
```

Comment:
There are already existing variables `M`, `I` etc going on in this file.  Can you use them rather than defining new ones within the lemma?